appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js

```diff
@@ -12,7 +12,8 @@ import { appealShortReference } from '#lib/appeals-formatter.js';
 export const changeSiteOwnershipPage = (appealData, appellantCaseData, storedSessionData) => {
 	const shortAppealReference = appealShortReference(appealData.appealReference);
 
-	let isFullyOwned = appellantCaseData.siteOwnership.isFullyOwned;
+	let isFullyOwned = appellantCaseData.siteOwnership.ownsAllLand;
```

Comment:
It doesn't look like these are being reassigned so they could be `const`

Reply:
+1

Reply:
Lines 19 and 20 if there is data in the session

Reply:
I can't see `isPrtiallyOwned` being reassigned, but it's really a trivial issue, to happy to resolve this point